config/locales/en/providers.yml

```diff
@@ -136,7 +136,7 @@ en:
         what_happens_next: What happens next
         info: You can return to this application when you're ready to make a substantive application.
         warning:
-          application_deadline_date: You must submit a substantive application by 10 November 2019.
+          application_deadline_date: "You must submit a substantive application by "
```

Comment:
Are the `"`s necessary (Genuine question, I don't know! It looks like some translations use them, some don't. Is it for interpolation?)